File: WOT-iOS/WOTData/WOTData/model/subclasses/Vehicleprofile+CoreDataProperties.m
//
//  Vehicleprofile+CoreDataProperties.m
//  WOTData
//
//  Created by Pavel Yeshchyk on 8/28/18.
//  Copyright © 2018 Pavel Yeshchyk. All rights reserved.
//
//

#import "Vehicleprofile+CoreDataProperties.h"

@implementation Vehicleprofile (CoreDataProperties)

+ (NSFetchRequest<Vehicleprofile *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Vehicleprofile"];
}

@dynamic hashName;
@dynamic hp;
@dynamic hull_hp;
@dynamic hull_weight;
@dynamic is_default;
@dynamic max_ammo;
@dynamic max_weight;
@dynamic speed_backward;
@dynamic speed_forward;
@dynamic tank_id;
@dynamic weight;
@dynamic ammo;
@dynamic armor;
@dynamic engine;
@dynamic gun;
@dynamic radio;
@dynamic suspension;
@dynamic tank;
@dynamic turret;

@end

File: WOT-iOS/Vehicleprofile+CoreDataProperties.swift
//
//  Vehicleprofile+CoreDataProperties.swift
//  WOTData
//
//  Created by Pavel Yeshchyk on 4/20/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

import Foundation
import CoreData

extension Vehicleprofile {
    @nonobjc public class func fetchRequest() -> NSFetchRequest<Vehicleprofile> {
        return NSFetchRequest<Vehicleprofile>(entityName: "Vehicleprofile")
    }

    @NSManaged public var hashName: NSDecimalNumber?
    @NSManaged public var hp: NSDecimalNumber?
    @NSManaged public var hull_hp: NSDecimalNumber?
    @NSManaged public var hull_weight: NSDecimalNumber?
    @NSManaged public var is_default: NSNumber?
    @NSManaged public var max_ammo: NSDecimalNumber?
    @NSManaged public var max_weight: NSDecimalNumber?
    @NSManaged public var profile_id: NSDecimalNumber?
    @NSManaged public var speed_backward: NSDecimalNumber?
    @NSManaged public var speed_forward: NSDecimalNumber?
    @NSManaged public var tank_id: NSDecimalNumber?
    @NSManaged public var weight: NSDecimalNumber?
    @NSManaged public var ammo: VehicleprofileAmmoList?
    @NSManaged public var armor: VehicleprofileArmorList?
    @NSManaged public var engine: VehicleprofileEngine?
    @NSManaged public var gun: VehicleprofileGun?
    @NSManaged public var modules: VehicleprofileModule?
    @NSManaged public var modulesTree: ModulesTree?
    @NSManaged public var radio: VehicleprofileRadio?
    @NSManaged public var suspension: VehicleprofileSuspension?
    @NSManaged public var turret: VehicleprofileTurret?
    @NSManaged public var vehicles: Vehicles?
}
